### DISCUSSION TITLE
Remove sensitive data globally

### DISCUSSION THREAD

**Discussion by akshah123** [reidmorrison/semantic_logger#272] [Q&A]
Hi,

I am wondering if there is a way to setup something that would remove certain sensitive information from all logs.  This would be useful if lets say you wanted to remove/redact Authorization header for incoming AND outgoing requests. 

Is this something already supported by this gem? If not, how would I go about it?

Thanks.

**Comment by reidmorrison** [comment]
This gem has no direct support, since it relies on the features already built into Rails for filtering out sensitive fields from the logs.
For example:
~~~ruby
Rails.application.config.filter_parameters += [:email, :phone]
~~~
